test/core/binary.wast

```diff
@@ -218,15 +129,6 @@
   "integer representation too long"
 )
 
-;; Unsigned LEB128 must not be overlong
-(assert_malformed
-  (module binary
-    "\00asm" "\01\00\00\00"
-    "\05\08\01"                          ;; Memory section with 1 entry
-    "\00\82\80\80\80\80\00"              ;; no max, minimum 2 with one byte too many
-  )
-  "integer representation too long"
-)
 
 ;; Signed LEB128 must not be overlong
```

Comment:
And these?

Reply:
Oh, right, those are already duplicated. Updated this PR to remove them as well.